prow/config/jobs/test-infra.yaml

```diff
@@ -31,62 +31,6 @@ jobs:
     command: [entrypoint, make, -C, authentikos, integ-test]
     requirements: [kind, gcp]
 
-  - name: deploy-prow
-    types: [postsubmit]
-    regex: '^prow/cluster/(?:gcsweb/|velodrome/|monitoring/|build/|private/|[^/]+\.yaml$)'
-    reporter_config:
-      slack:
-        channel: "oncall"
-        job_states_to_report:
-        - success
-        - failure
-        - aborted
-        - error
-        report_template: 'Deploying prow: {{.Status.State}}. <{{.Spec.Refs.BaseLink}}|Commit {{.Spec.Refs.BaseSHA}}> <{{.Status.URL}}|View logs> <https://testgrid.k8s.io/istio_test-infra_postsubmit#deploy-prow_test-infra_postsubmit|Job history on Testgrid>'
-    cluster: test-infra-trusted
-    max_concurrency: 1
-    command:
-    - make
-    - -C
-    - prow
-    - deploy
-    - deploy-gcsweb
-    - deploy-velodrome
-    - deploy-monitoring
-    - deploy-build
-    - deploy-private
-    requirements: [deploy]
-    node_selector:
-        prod: prow
-
-  - name: deploy-boskos
-    types: [postsubmit]
-    regex: '^boskos/cluster/.*\.yaml$'
-    cluster: test-infra-trusted
-    max_concurrency: 1
-    command:
-    - make
-    - -C
-    - boskos
-    - deploy
-    requirements: [deploy]
-    node_selector:
-        prod: prow
-
-  - name: deploy-boskos-config
```

Comment:
Why did we move it? This may limit our auto-image updater, etc.

if there are limitations in the job generation we should address those IMO. If its just serviceAccountName missing we will probably need that anyways for our other (istio dev team owned) jobs currently using SA keys

Reply:
I moved these jobs because they are infrastructure jobs that should be managed by our oncall team. We're not really familiar with Istio's job generator so I think it makes more sense for these jobs to be defined directly in Prow config like we're used to. Even if the job generation is updated to support serviceAccountName, I'd prefer to avoid the extra generation step; I don't think it is really adding value for these 3 jobs.

> This may limit our auto-image updater, etc.

Could you elaborate on this concern please? I wouldn't expect this to change to cause problems with auto image bumping, directly defined jobs should also be bumped. If that is not the case then that seems like the area to fix to me.